src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -216,7 +216,7 @@ protected function prepareException(Exception $e)
     protected function unauthenticated($request, AuthenticationException $exception)
     {
         return $request->expectsJson()
-                    ? response()->json(['message' => 'Unauthenticated.'], 401)
+                    ? response()->json(['message' => $exception->getMessage()], 401)
```

Comment:
Probably should be `$exception->getMessage() ?: 'Unauthenticated.'`?

Reply:
I think it's not necessary because the constructor for AuthenticationException already has a default value for the $message property

```php
public function __construct($message = 'Unauthenticated.', array $guards = [])
{
    parent::__construct($message);

    $this->guards = $guards;
}
```

Reply:
> I think it's not necessary because the constructor for AuthenticationException already has a default value for the $message property

And what if someone actuallty constructs it with somethinh empty?

Reply:
Something empty like:

`throw new AuthenticationException('')` ?

If this is the case I would expect that this person does not want any message at all.

Or you were thinking of something else?

Reply:
I'm with @aocneanu on this.

Reply:
Also, while inspecting a little more I found that this exception is used in 2 middlewares and one trait:

- `Authenticate` middleware
Here the exception is thrown using "Unauthenticated." message.

- `AuthenticateSession` middleware
thrown without a message.

- `GuardHelpers` trait
thrown without a message.

If we want to be consistent we could either

a) hardcode the "Unauthenticated." message in `AuthenticateSession` middleware and in `GuardHelpers` trait, and maybe remove the default message from the `AuthenticationException` constructor. In this case, in `Handler` we could put the an elvis operator if we consider that the user never wants to throw this exception without a message, or a ternary and cover both situations

b) remove the "Unauthenticated." message from the `Authenticate` middleware and keep the default value in the `AuthenticationException` constructor.

If you find any of this suitable let me know and I will commit again.